bin/dcso_tie_setup_handler.py

```diff
@@ -3,9 +3,23 @@
 # This file is based on the skeleton Python script provided by Splunk.
 # It is adapted to be more conform to Python code styling.
 
+import os
+import sys
+
 import splunk.admin as admin
 # import splunk.entity as en
 
+try:
+    import dcsotie
+except ImportError:
+    sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "lib"))
```

Comment:
These not-so nice path inserts are needed because of running within the Splunk environment. The app/add-on is not "installed", but placed in an `apps/` folder. What you are sure of is that the `bin/` folder is a peer of `lib/`.